lib/board.rb

```diff
@@ -0,0 +1,60 @@
+class Board 
+  attr_accessor :cells 
+  
+  
+  def initialize
+    @cells = Array.new(9," ")
```

Comment:
I would use this implementation in the `reset!` method, then call that method from `initialize`.